webapp/service/medlemService.ts

```diff
@@ -0,0 +1,56 @@
+const API_URL = 'https://api.ntnui.no'
```

Comment:
This could be an optional environment variable maybe? It is built into React, so I think you can just
```suggestion
const API_URL = process.env.REACT_APP_API_URL || 'https://api.ntnui.no'
```

And add it to the table in the README if you do, together with other relevant info for further development and testing, such as dealing with CORS 😛